src/xdocs/checks/annotation/annotationusestyle.xml

```diff
@@ -130,116 +130,139 @@
         </div>
       </subsection>
       <subsection name="Examples" id="Examples">
-        <p> To configure the check:</p>
+        <p id="Example1-config">
+          To configure the check:
+        </p>
         <source>
-&lt;module name=&quot;AnnotationUseStyle&quot;/&gt;
+&lt;module name=&quot;Checker&quot;&gt;
+  &lt;module name=&quot;TreeWalker&quot;&gt;
+    &lt;module name=&quot;AnnotationUseStyle&quot;/&gt;
+  &lt;/module&gt;
+&lt;/module&gt;
         </source>
-        <p>
-          Example:
-        </p>
+        <p id="Example1-code">Example:</p>
         <source>
-@SuppressWarnings(&quot;unchecked&quot;) // OK
-@Deprecated // OK
-@SomeArrays({&quot;unchecked&quot;,&quot;unused&quot;}) // OK
-public class TestOne
+@SuppressWarnings(&quot;unchecked&quot;) // ok as it's in implied style
+@Deprecated // // ok as it matches closingParens default property
```

Comment:
remove extra `//`

Reply:
My bad. Will keep such typos under check. Done.

Reply:
Also, @romani for future PRs, shouldn't we have complete explanation of // OK as we are already cleaning up such statements along with their suppressions. Shouldn't having an // OK with some additional context be a standard practice?

Reply:
it depends.
Sometime code is so self-explanatory, so extra comment is not required and even make a noise for user during reading.

In this PR, I think it is beneficial to have some comments.

Reply:
Thank you for the insights.